back-end/src/main/kotlin/Gneiss/PacketCompiler/Service/PacketRequestHandler.kt

```diff
@@ -39,10 +44,17 @@ class ApprovalPDFPostResponse()
 
 class InvoicePDFPostResponse()
 
-class PacketRequestHandler(pdfHelper: IPDFHelper, packetDao: IPacketDao) {
+class PacketGetResponse(
```

Comment:
We'll be making a Get for a single packet soon, so maybe this could be called `PacketsGetResponse`